src/ILCompiler.Compiler/src/CppCodeGen/CppWriter.cs

```diff
@@ -93,6 +90,21 @@ private void BuildExternCSignatureMap()
             }
         }
 
+        private IEnumerable<string> GetParameterNamesForMethod(MethodDesc method)
+        {
+            // TODO: The uses of this method need revision. The right way to get to this info is from
+            //       a MethodIL. For declarations, we don't need names.
```

Comment:
(At this point I'm considering taking parameter names handling out of `MethodDebugInformation` and making it a completely separate beast, but I was hoping I could abstract this weirdness out. By weirdness I mean storing some of the symbolic information within the runtime image, and some in the PDB.)